proto/eth/v1/beacon_chain.proto

```diff
@@ -339,3 +339,20 @@ message ForkChoiceNode {
     // Timestamp when the node was inserted
     uint64 timestamp = 12;
 }
+
+message BlobsRequest {
+    // The block identifier. Can be one of: "head" (canonical head in node's view), "genesis",
+    // "finalized", <slot>, <hex encoded blockRoot with 0x prefix>.
+    bytes block_id = 1;
+}
+
+message Blob {
+    bytes data = 1;
+}
+
+message BlobsResponse {
+    bytes beacon_block_root = 1;
+    uint64 beacon_block_slot = 2;
+    repeated Blob blobs = 3;
```

Comment:
nvmind. I see that the `BlobsSidecar` doesn't group blobs by transaction ordering either.

Reply:
Chatted offline... this would require a refactor of the blob storage, because blobs are stored in the sidecar as an array, not grouped by transaction.